lib/initialize.js

```diff
@@ -6,11 +6,11 @@ async function initialize(knex, modelClasses) {
     knex = modelClasses[0].knex();
   }
 
-  for (const modelClass of modelClasses) {
-    await modelClass.fetchTableMetadata({
-      knex
-    });
-  }
+  const promises = modelClasses.reduce((acc, modelClass) => {
```

Comment:
Isn't `.map` more appropriate in this case?

Reply:
yes, doh.  Thanks.